### ISSUE TITLE
Forms: Cannot read property 'enabled' of null

### ISSUE THREAD

**Issue by Fank** [angular/angular#11535]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior** 
Will throw an "EXCEPTION" on adding a `FormControl` to an empty `FormGroup`

**Expected behavior**
Do not throw

**Reproduction of the problem**
I think the following example will explain everything.
http://plnkr.co/edit/MBNJDjYd3GZiT9g7RXil?p=preview
1. Click the "Add" button
2. See that the error will be thrown

**What is the motivation / use case for changing the behavior?**
-

**Please tell us about your environment:**
-

* **Angular version:** 2.0.0-rc.6
Tested with rc.6

* **Browser:** [all] 
 
* **Language:** [all]
 
* **Node (for AoT issues):** `node --version` =   6.5.0

**Comment by Fank** [comment]
I found the problem, its caused because the default "delimiter" in the `_find` function is `.`.
And the control name is a email address with `.` in the name, so the control name will be taken as a path.

**Comment by kara** [comment]
Thanks, @Fank. Should be fixed when #11542 lands.

**Comment by Fank** [comment]
@kara you are welcome
